src/components/PostCard/PostCard.test.tsx

```diff
@@ -23,4 +25,49 @@ describe("Given the PostCard component", () => {
       expect(title).toBeInTheDocument();
     });
   });
+
+  describe("When it receives post with a author: 'Macario'", () => {
```

Comment:
whe it receives a post with the author/  with an author

Reply:
cambiado